src/Collection.php

```diff
@@ -936,6 +937,23 @@ public function updateOne($filter, $update, array $options = [])
         return $operation->execute($server);
     }
 
+    /*
+     * ChangeStream outline
+     *
+     * @see ChangeStream::__construct() for supported options
+     * @param array          $pipeline       List of pipeline operations
+     * @param array          $options        Command options
+     * @throws InvalidArgumentException for parameter/option parsing errors
+     */
+    public function watch(array $pipeline, array $options = [])
+    {
+        $server = $this->manager->selectServer($this->readPreference);
```

Comment:
Much like other CRUD methods, we can take a `readPreference` option in `$options` and fall back to the Collection's read preference if it's not specified. The CRUD and ChangeStream spec don't require per-operation options for read preference, read concern, and write concern, but PHPLIB does it throughout (mostly for historical reasons).

We can do the same for `readConcern`. The current spec doesn't allow that (it requires we inherit the Collection's read concern), but I believe that was an oversight. I've opened https://github.com/mongodb/specifications/pull/237 to correct that.